Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM node:16 AS builder
+
+WORKDIR /app
+
+RUN apt-get update -qq && apt-get install -y build-essential
+RUN npm install -g sass
+COPY package.json /app/package.json
+COPY yarn.lock /app/yarn.lock
+
+
+COPY src /app/src
+COPY public /app/public
+
+RUN yarn install
+RUN yarn build
+
+
+
+
+
+FROM ubuntu:latest
+
+USER root
+
+RUN apt-get update
+RUN apt-get install -y nginx
+
+# Remove the default Nginx configuration file
+RUN rm -v /etc/nginx/nginx.conf
+
+# Copy a configuration file from the current directory
+ADD nginx.conf /etc/nginx/
+
+COPY --from=builder  /app/build/ /usr/share/nginx/html/
+COPY --from=builder  /app/build/ /var/www/html/
+
+# Append "daemon off;" to the beginning of the configuration
+RUN echo "daemon off;" >> /etc/nginx/nginx.conf
+
+# Expose ports
+EXPOSE 90
```

Comment:
90?

Reply:
it's internal (you map it when you instantiate the container), I will change it to be aligned with other containers of the project. Thanks for pointing it.

Reply:
FYI, the Django container use port 9000